multisig/proposals/ethereum/vip-035/index.ts

```diff
@@ -0,0 +1,162 @@
+export const CURVE_XVS_DISTRIBUTOR = "0x8473B767F68250F5309bae939337136a899E43F9";
+export const LST_XVS_DISTRIBUTOR = "0x7A91bEd36D96E4e644d3A181c287E0fcf9E9cc98";
+
+export const SPEEDS = [
+  {
+    name: "vweETH",
+    address: CORE_vwETH,
+    reward: parseUnits("1125", 18),
+    supplySpeedPercentage: 40,
+    borrowSpeedPercentage: 60,
+    distributor: CORE_XVS_DISTRIBUTOR,
+  },
+  {
+    name: "vWBTC",
+    address: CORE_vWBTC,
+    reward: parseUnits("3375", 18),
+    supplySpeedPercentage: 40,
+    borrowSpeedPercentage: 60,
+    distributor: CORE_XVS_DISTRIBUTOR,
+  },
+  {
+    name: "vUSDT",
+    address: CORE_vUSDT,
+    reward: parseUnits("3375", 18),
+    supplySpeedPercentage: 40,
+    borrowSpeedPercentage: 60,
+    distributor: CORE_XVS_DISTRIBUTOR,
+  },
+  {
+    name: "vUSDC",
+    address: CORE_vUSDC,
+    reward: parseUnits("3375", 18),
+    supplySpeedPercentage: 40,
+    borrowSpeedPercentage: 60,
+    distributor: CORE_XVS_DISTRIBUTOR,
+  },
+  {
+    name: "vcrvUSD",
+    address: CORE_vcrvUSD,
+    reward: parseUnits("1500", 18),
+    supplySpeedPercentage: 40,
+    borrowSpeedPercentage: 60,
+    distributor: CORE_XVS_DISTRIBUTOR,
+  },
+  {
+    name: "vFRAX",
+    address: CORE_vFRAX,
+    reward: parseUnits("600", 18),
+    supplySpeedPercentage: 40,
+    borrowSpeedPercentage: 60,
+    distributor: CORE_XVS_DISTRIBUTOR,
+  },
+  {
+    name: "vsFRAX",
+    address: CORE_vsFRAX,
+    reward: parseUnits("600", 18),
+    supplySpeedPercentage: 40,
+    borrowSpeedPercentage: 60,
+    distributor: CORE_XVS_DISTRIBUTOR,
+  },
+  {
+    name: "vTUSD",
+    address: CORE_vTUSD,
+    reward: parseUnits("200", 18),
+    supplySpeedPercentage: 40,
+    borrowSpeedPercentage: 60,
+    distributor: CORE_XVS_DISTRIBUTOR,
+  },
+  {
+    name: "vDAI",
+    address: CORE_vDAI,
+    reward: parseUnits("500", 18),
+    supplySpeedPercentage: 40,
+    borrowSpeedPercentage: 60,
+    distributor: CORE_XVS_DISTRIBUTOR,
+  },
+  {
+    name: "vCRV",
+    address: CURVE_vCRV,
+    reward: parseUnits("375", 18),
+    supplySpeedPercentage: 40,
+    borrowSpeedPercentage: 60,
+    distributor: CURVE_XVS_DISTRIBUTOR,
+  },
+  {
+    name: "vcrvUSD",
+    address: CURVE_vcrvUSD,
+    reward: parseUnits("375", 18),
+    supplySpeedPercentage: 40,
+    borrowSpeedPercentage: 60,
+    distributor: CURVE_XVS_DISTRIBUTOR,
+  },
+  {
+    name: "vWETH",
+    address: LST_vwETH,
+    reward: parseUnits("18333", 18),
+    supplySpeedPercentage: 30,
+    borrowSpeedPercentage: 70,
+    distributor: LST_XVS_DISTRIBUTOR,
+  },
+  {
+    name: "vwstETH",
+    address: LST_vwstETH,
+    reward: parseUnits("3600", 18),
+    supplySpeedPercentage: 100,
+    borrowSpeedPercentage: 0,
+    distributor: LST_XVS_DISTRIBUTOR,
+  },
+  {
+    name: "vweETH",
+    address: LST_vweETH,
+    reward: parseUnits("0", 18),
+    supplySpeedPercentage: 40,
+    borrowSpeedPercentage: 60,
+    distributor: LST_XVS_DISTRIBUTOR,
+  },
```

Comment:
I think we don't need to add this, because there are no changes for the rewards on the `vweETH` market